sqlx-core/src/mysql/options.rs

```diff
@@ -120,6 +121,7 @@ impl MySqlConnectOptions {
             database: None,
             ssl_mode: MySqlSslMode::Preferred,
             ssl_ca: None,
+            statement_cache_size: 100,
```

Comment:
I decided this to be a good default. Up for discussion.

Reply:
`100` sounds fine to me. I don't have any context for that though. I figure just pick a number out of a hat or profile to find a reasonable size for a server with a lot of dynamic queries. We definitely want the cache on by default.